web/client/src/library/components/editor/EditorInspector.tsx

```diff
@@ -377,68 +381,88 @@ function FormActionsModel({
               </Banner>
             </FormFieldset>
           )}
-          <fieldset className="my-3 px-3">
+          <fieldset className="my-3 px-3 w-full">
             <Input
               className="w-full mx-0"
-              size={EnumSize.sm}
               label="Start Date"
-              placeholder="02/11/2023"
-              value={form.start}
-              onInput={(e: React.ChangeEvent<HTMLInputElement>) => {
-                e.stopPropagation()
-
-                setForm({
-                  ...form,
-                  start: e.target.value ?? '',
-                })
-              }}
-            />
+            >
+              {({ className }) => (
+                <Input.Textfield
+                  className={clsx(className, 'w-full')}
+                  placeholder="02/11/2023"
```

Comment:
Probably outside the scope of this PR, but should these date fields use YYYY-MM-DD format?